examples/keyword-extraction-api.ipynb

```diff
@@ -0,0 +1,615 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "xU1WBQJ7Uh09",
+   "metadata": {
+    "id": "xU1WBQJ7Uh09"
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b8d580f8-67d5-45d6-915f-0f6b60d3b543",
+   "metadata": {},
+   "source": [
+    "Input your unique kluster.ai API key. If you haven’t obtained one yet, don’t forget to <a href=\"https://platform.kluster.ai/signup\" target=\"_blank\">sign up</a>."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "4911e5eb-7463-4a6b-8a99-9313cecd9d4d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdin",
+     "output_type": "stream",
+     "text": [
+      "Enter your kluster.ai API key:  ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "from getpass import getpass\n",
+    "\n",
+    "api_key = getpass(\"Enter your kluster.ai API key: \")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bcc3d475-8f49-4fc4-9a5e-c6eb6866d2b5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install -q openai"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "b89a4feb-37a9-430d-a742-cd58495b4eaf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openai import OpenAI\n",
+    "\n",
+    "import pandas as pd\n",
+    "import time\n",
+    "import json\n",
+    "from IPython.display import clear_output, display"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "zG9y_WO5rYaj",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up the client\n",
+    "client = OpenAI(\n",
+    "    base_url=\"https://api.kluster.ai/v1\",\n",
+    "    api_key=api_key,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "udPtLfTaisSw",
+   "metadata": {
+    "id": "udPtLfTaisSw"
+   },
+   "source": [
+    "## Get the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "QjCVfg65jKz6",
+   "metadata": {
+    "id": "QjCVfg65jKz6"
+   },
+   "source": [
+    "This notebook comes with a preloaded sample dataset based on the AG News dataset. It includes excerpts of news headlines and their leads, all set for processing. There’s no extra setup required—just move on to the next steps to start working with the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "07018f92-9a01-47d5-916a-12cd03dfa3a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame({\n",
+    "    \"text\": [\n",
+    "        \"Chorus Frog Found Croaking in Virginia - The Southern chorus frog has been found in southeastern Virginia, far outside its previously known range. The animal had never before been reported north of Beaufort County, N.C., about 125 miles to the south.: nan\",\n",
```

Comment:
What's the meaning of "nan" in this data record? From the description that this is an excerpt from a headline and a lead, I was expecting this to be the lead.